source/options.ts

```diff
@@ -73,14 +84,14 @@ export const buildOptions = (helpText, options) => {
 	const pkg = foundPackage ?? {};
 	normalizePackageData(pkg);
 
-	const parsedOptions = {
+	const parsedOptions: ParsedOptions = {
 		argv: process.argv.slice(2),
 		flags: {},
 		inferType: false,
-		input: 'string',
+		input: 'string', // TODO: undocumented option?
```

Comment:
Should this option be documented?

Reply:
No, it doesn't really make sense to define a different type for all arguments.